code/obj/item/storage/backpack_belt_etc.dm

```diff
@@ -1113,6 +1113,21 @@ TYPEINFO(/obj/item/storage/belt/wrestling)
 	is_syndicate = 0
 	fake = 1
 
+/obj/item/storage/belt/wrestling/hos
+	name = "Head of Security's championship belt"
+	desc = "The Head of Security's prized wrestling belt."
+	contraband = 6
+
+	can_hold = list(/obj/item/baton, // Only include the things the HoS will be keeping on them.
+	/obj/item/gun/energy/taser_gun,
+	/obj/item/gun/energy/egun,
+	/obj/item/gun/energy/lawbringer,
+	/obj/item/gun/energy/wavegun,
+	/obj/item/gun/energy/tasersmg,
+	/obj/item/gun/energy/tasershotgun,
+	/obj/item/device/prisoner_scanner)
```

Comment:
less dupe pls (could still add tasershotgun in New like the shotgun belt does i guess)
```suggestion
	can_hold = /obj/item/storage/belt/security::can_hold
```